repository-data/webfiles/src/main/resources/site/freemarker/hee/layout/base-breadcrumb.ftl

```diff
@@ -0,0 +1,25 @@
+<#include "../../include/imports.ftl">
+
+<#-- @ftlvariable name="breadcrumbLinks" type="java.util.List<uk.nhs.hee.web.components.beans.BreadcrumbLink>" -->
+<#if breadcrumbLinks?? && breadcrumbLinks?has_content>
+    <@hst.setBundle basename="uk.nhs.hee.web.global"/>
+
+    <nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+        <div class="nhsuk-width-container">
+            <#-- For Desktop -->
+            <ol class="nhsuk-breadcrumb__list">
+                <#list breadcrumbLinks as breadcrumbLink>
+                    <li class="nhsuk-breadcrumb__item">
+                        <a class="nhsuk-breadcrumb__link" href="${breadcrumbLink.url}">${breadcrumbLink.text?html}</a>
```

Comment:
@thojava - Thanks for the info.

Looks like the recommended way to enable auto escaping is to setup output formatting and Freemarker recommends (https://freemarker.apache.org/docs/pgui_config_outputformatsautoesc.html) to use `.ftlh` file extension to auto set `HTML` output formatting, but I ended up with the following `FileNotFoundException` error (though I tried setting up `incompatible improvements` setting described in https://freemarker.apache.org/docs/pgui_config_incompatible_improvements.html#pgui_config_incompatible_improvements_how_to_set):
```
[INFO] [talledLocalContainer] 05.03.2021 12:18:41 WARN  http-nio-8080-exec-6 [DefaultPageErrorHandler.logWarningsForEachComponentExceptions:49] Component exception on uk.nhs.hee.web.components.BreadcrumbComponent
[INFO] [talledLocalContainer] java.io.FileNotFoundException: The requested resource [/site/freemarker/hee/layout/base-breadcrumb.ftlh] is not available
[INFO] [talledLocalContainer] 	at org.apache.catalina.servlets.DefaultServlet.serveResource(DefaultServlet.java:845) ~[catalina.jar:9.0.37]
```
Not sure, probably Bloomreach need to add the support for `.ftlh` extension based freemarker templates internally.

Have enabled `HTML` output formatting via the following `ftl` tag (which enables auto escaping):
```
<#ftl output_format="HTML">
```

Fyi - Also, have refactored `repository-data/webfiles/src/main/resources/site/freemarker/include/imports.ftl` so that it contains the common tags (e.g. hst & fmt) and `locale` & `url_escaping_charset` freemarker settings which I think we can readily include on  other templates if required. Thanks!